recipes/dipy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.11.0" %}
+
+package:
+  name: dipy
+  version: {{ version }}
+
+source:
+  fn: dipy-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/d/dipy/dipy-{{ version }}.tar.gz
+  md5: 7723e8994d31e910d47b7a0cf064a3d8
+
+build:
+    number: 0
+    script: python setup.py install
+    features:
+        - vc9  # [win and py27]
+        - vc10  # [win and py34]
+        - vc14  # [win and py35]
```

Comment:
I may have been mistaken.  These are not needed in order to compile with Cython on windows?